src/main/java/com/exasol/adapter/dialects/rls/RowLevelSecurityQueryRewriter.java

```diff
@@ -35,51 +35,50 @@ public String rewrite(final SqlStatement statement, final ExaMetadata exaMetadat
           final AdapterProperties properties) throws AdapterException, SQLException {
         if (statement instanceof SqlStatementSelect) {
             final SqlStatementSelect select = (SqlStatementSelect) statement;
-            final SqlStatementSelect.Builder rslStatementBuilder =
+            final SqlStatementSelect.Builder rlsStatementBuilder =
                   SqlStatementSelect.builder().selectList(select.getSelectList()).fromClause(select.getFromClause());
             if (select.hasGroupBy()) {
-                rslStatementBuilder.groupBy(select.getGroupBy());
+                rlsStatementBuilder.groupBy(select.getGroupBy());
             }
             if (select.hasHaving()) {
-                rslStatementBuilder.having(select.getHaving());
+                rlsStatementBuilder.having(select.getHaving());
             }
             if (select.hasLimit()) {
-                rslStatementBuilder.limit(select.getLimit());
+                rlsStatementBuilder.limit(select.getLimit());
             }
             if (select.hasOrderBy()) {
-                rslStatementBuilder.orderBy(select.getOrderBy());
+                rlsStatementBuilder.orderBy(select.getOrderBy());
             }
-            this.applyWhereClause(select, rslStatementBuilder);
-            final SqlStatementSelect newSelectStatement = rslStatementBuilder.build();
+            applyWhereClause(select, rlsStatementBuilder);
+            final SqlStatementSelect newSelectStatement = rlsStatementBuilder.build();
             return super.rewrite(newSelectStatement, exaMetadata, properties);
         } else {
             throw new IllegalArgumentException(
-                  "Modified SQL statement must be a select statement, but was " + statement.getClass().getName());
+                  "Modified SQL statement must be a SELECT statement, but was " + statement.getClass().getName());
         }
     }
 
     private void applyWhereClause(final SqlStatementSelect select,
           final SqlStatementSelect.Builder rslStatementBuilder) {
-        final UserInformation userInformation = new UserInformation();
+        final UserInformation userInformation = new UserInformation("rls_users");
```

Comment:
```suggestion
        final UserInformation userInformation = new UserInformation("exa_rls_users");
```

To avoid namespace collisions (even if they are very unlikely).

Reply:
done